lib/ts/station_fsm.ex

```diff
@@ -4,16 +4,22 @@ defmodule StationFsm do
 	"""
 
 	use Fsm, initial_state: :start, initial_data: []
+	require Itinerary
 
 	# Function definitions
+	def initialise_fsm(input = [_station_vars,
```

Comment:
why are the underscore prefix used for the variables here?

Reply:
It is convention to add an underscore to placeholder variables.